### DISCUSSION TITLE
Feature request: window open

### DISCUSSION THREAD

**Discussion by MM502** [apohl79/audiogridder#1093] [Ideas]
It would be helpful to set a preference for AG to open the Statistics window (server) and Monitor window (plug-in) and keep them open. 
This is especially true for the plug-in which closes the Monitor window when clicking on it. 
It would also be useful for the server's Statistics window as it closes every time the server is restarted.

**Comment by rncwalker** [comment]
It also should allow the user to move it to where they want as a global setting and place an 'X' for the user to close if they want.  many of us have a multi monitor setups.

Robert

**Comment by apohl79** [comment]
Thanks for the suggestions! I created feature requests for them.
